script.js

```diff
@@ -1,228 +1,295 @@
+      <li class="css">${array[0].tech[1]}</li>
+      <li class="javascript">${array[0].tech[2]}</li>
+      <li class="html">${array[0].tech[3]}</li>
+      <li class="html">${array[0].tech[4]}</li>
+      <li class="html">${array[0].tech[5]}</li>
     </ul>
-    <div class ="lorem"><h2>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</h2></div>
+    <div class ="lorem">${array[0].desc}</div>
+    <div class ="frame19"><button class="mobile-first-button1" type="button" id="messagep">See source <img class="btn-img" src="${array[0].githubImg}" alt=""></button>
+       <button class="mobile-first-button2" type="button" id="messagep">See live <img class="btn-img" src="${array[0].liveImg}" alt=""></button></div>
+    
   `;
-  sectioncard.appendChild(main);
-  sectioncard.appendChild(maincard);
+
+  main.appendChild(maincard);
+  body.appendChild(main);
 
   const closeButton2 = document.querySelector('.close-button2');
   closeButton2.addEventListener('click', () => {
-    sectioncard.removeChild(maincard);
-    sectioncard.removeChild(main);
+    main.removeChild(maincard);
+    body.removeChild(main);
   });
 });
 
-const sectioncard2 = document.querySelector('.project-2');
 const openerButton2 = document.querySelector('.button2');
 openerButton2.addEventListener('click', () => {
   const main = document.createElement('div');
   main.className = 'main';
   const maincard = document.createElement('div');
   maincard.className = 'maincard';
   maincard.innerHTML = `
+
   <button class="close-button2">&times;</button>
   <div>
  
-  <img  class = "popup-image"src="images/pop-up/Snapshoot Portfolio.png" alt="">
+  <img  class = "popup-image"src="${array[0].mobileImg}" alt="">
+  <img  class = "popup-image2"src="${array[0].desktopImg}" alt="">
  </div>
-       <div class ="frame18"> <button class="first-button1" type="button" id="messagep">See source <img src="images/pop-up/button-icons/github.png" alt=""></button>
-        <button class="first-button2" type="button" id="messagep">See live <img src="images/pop-up/button-icons/Icon.png" alt=""></button>
-    <h3 class="popup-par">Keeping track of hundreds of <br> components</h3></div>
+ <div class="headline">
+    <h2 class="popup-par">${array[0].name}</h2>
+    <div class ="frame18">
+       <button class="first-button2" type="button" id="messagep">See live <img src="${array[0].liveImg}" alt=""></button>
+       <button class="first-button1" type="button" id="messagep">See source <img src="${array[0].githubImg}" alt=""></button>
+    </div>
+  </div>   
     <ul class="popup-list">
```

Comment:
- [ ] Good job so far, but as mentioned by the last reviewer your pop-up windows is still cut off the bottom text and information. Please fix it in order to make your project works as expected 👍

I think you should remove the height property that you added to the main class
![image](https://user-images.githubusercontent.com/63473584/226144608-3f6c1987-df42-4614-9fc7-eabd764741aa.png)

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/226144631-81ce3850-6dff-4244-8530-c6db2a8ae16e.png)

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/226144640-d49c1489-993b-4773-93ae-b26c09e5d449.png)